covid_api/db/utils.py

```diff
@@ -13,10 +13,10 @@ def s3_get(bucket, key):
     response = s3.get_object(Bucket=bucket, Key=key)
     return response["Body"].read()
 
-def get_indicator_notes(identifier: str, folder:str) -> Union[str, None]:
+def get_indicator_site_metadata(identifier: str, folder:str) -> Union[Dict, None]:
     try:
-        key = f"indicators/{folder}/{identifier}.txt"
-        return s3_get(INDICATOR_BUCKET, key).strip()
+        key = f"indicators/{folder}/{identifier}.json"
+        return json.loads(s3_get(INDICATOR_BUCKET, key))
     except Exception:
         return None
```

Comment:
@drewbo Might be safer to return `{}` here, because later you are doing `site_metadata.get('notes', None),`